src/openrct2/ride/Ride.cpp

```diff
@@ -2635,6 +2636,11 @@ static void ride_mechanic_status_update(Ride* ride, int32_t mechanicStatus)
                 ride->window_invalidate_flags |= RIDE_INVALIDATE_RIDE_MAINTENANCE;
                 ride_mechanic_status_update(ride, RIDE_MECHANIC_STATUS_CALLING);
             }
+            // if the ride is broken down, but a mechanic was heading for an inspection, update orders to fix
+            else if (mechanic->State == PeepState::HeadingToInspection && !forInspection)
+            {
+                ride_call_mechanic(ride, mechanic, forInspection);
```

Comment:
```suggestion
                ride_call_mechanic(ride, mechanic, true);
```

Reply:
@IntelOrca should we also change the `forInspection` variable defined above to `forBreakdown`? I don't think changing this line on its own adds any additional clarity, as forInspection must be false to get into this conditional block.

Reply:
Rename the variable to whatever you think is clearest. But I thought it is clearer to pass a literal to the function since it will always be false. Ideally I would do `ride_call_mechanic(ride, mechanic, forInspection: false);`, but that is not possible in C++. That is the problem with boolean parameters, they don't tend to be as clear. An enum would be better or two distinct functions.

Reply:
@IntelOrca updated and added a few comments - hopefully it's clearer now